### PR TITLE
Add "Notice for use of battery powered beacons" page in the Quick Start Guide

### PR DESCRIPTION
**Pull request by itohtaka**

### CODE REVIEW

File: quick_start/battery.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+In cool environment, Lithium battery is better than alkaline battery.
+
+## Beacon settings and battery life
+Battery life of beacon will strongly depend on advertising interval and transmission power settings.
+
+Power consumption rate of 100ms advertising interval is 2 times of 200ms advertising interval.
+Increasing transmission power also consumes battery power.
+
+Advertising interval value is critical for beacon performance and it also affect user experience.
+Please conduct beacon vendor for optimum settings.
```
Comment (itoko) [inline quick_start/battery.md]: ”please conduct” => "please contact" (same typos are found in several lines)

File: quick_start/battery.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## When should you replace battery?
+Please replace battery when battery level becomes less then the following voltage even if beacon is still working. 
+
+| Type of Battery | Dead battery voltage |
+|--------|------|
+| CR2477 | 2.0V |
+| AA	 | 1.0V |
+
+Alkaline battery will cause electrolyte leakage if you leave it after dead voltage.
```
Comment (itoko) [inline quick_start/battery.md]: Leaving alkaline battery in dead voltage may cause electrolyte leakage.とかでどうでしょう？

**Comment by itohtaka** [comment]
Thank you very much for your review. 
I applied your suggestions. Please check it again.

File: quick_start/battery.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+| Type of battery | Capacity |
+|---------------------|---------|
+| 1 x CR2477 (Lithium)| 1000mAh |
+| 2 x AA (Alkaline)   | 2000mAh - 2700mAh (Manufacturer dependent) |
+| 2 x AA (Lithium)    | 3000mAh | 
+
+2 x AA (Lithium) powered beacon have 3 times battery life of CR2477 powered beacon.
+
+In cool environment, Lithium battery is better than alkaline battery.
```
Comment (itoko) [inline quick_start/battery.md]: cool -> cold would be better?

Comment (itoko) [inline quick_start/battery.md]: cool -> cold would be better?

File: quick_start/battery.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+   LED color changes on dead battery level.
+   Please contact beacon vendor for details.
+* Beacon management tools
+ * Some beacon vendor support battery level checker with special application program.
+   Please contact beacon vendor for details. 
+* Voltage meter
+ * If none of above indicator is available, you need to remove battery from beacon then use voltage meter to check battery level.
+* RSSI level  
+ * If it is very hard to remove battery, you can check RSSI level using general-purpose beacon check tools such as Beacon Ranging (https://itunes.apple.com/jp/app/beacon-ranging/id734155067). 
+   You can find beacons which already dead (Typically less than 0.7v on AA battery).
```
Comment (itoko) [inline quick_start/battery.md]: You can find beacons which are already dead (Typically less than 0.7V on AA battery). 

Excuse me for my tiny comments.

**Comment by itohtaka** [comment]
Thank you for corrections!
I updated battery.md file.
Best Regards

**Pull request merged**